### DISCUSSION TITLE
How long does it take to review my project?

### DISCUSSION THREAD

**Discussion by pctsvn** [WalletConnect/walletconnect-monorepo#2890] [Developer Support] [answered]
I have a live website using WalletConnect.
I have submitted review for my app.
My app is live so it's urgent for me to have my project published ASAP.
Could you please assist? 
Thanks!

**Comment by boidushya** [comment]
Hey! Projects pending review shouldn't have any outstanding issues as far as functionalities are concerned. Meanwhile, could you provide us with your projectId?

**Reply by pctsvn** [reply]
yes my app is fully working.
My project ID is 3b2702fff16f0a67ec37f565da4a5401.
Please help check it ASAP.
Thanks very much @boidushya !

**Reply by pctsvn** [reply]
any update please @boidushya ?

**Comment by chadyj** [comment, marked as answer]
You don't need your project approved to use WalletConnect. Just create a project and start connecting to web3.

The approval process is only for published [Explorer listings](https://walletconnect.com/explorer).

**Reply by pctsvn** [reply]
Upgrading Rainbowkit to version >= 1.0.2 fixes the issue.
Thanks @chadyj
